WebRoot/js/zimbraMail/share/view/dialog/ZmUploadDialog.js

```diff
@@ -243,6 +243,13 @@ function(files, uploadForm, folder) {
 	}
 };
 
+ZmUploadDialog.prototype.setFileExtensions = function(){
+    var extensionsData = appCtxt.getSettings().getInfoResponse.attrs._attrs.zimbraFeatureBriefcaseAllowedFileExtensions;
+    var array;
```

Comment:
can you use a better variable here?

Reply:
sure... changing 'array' into 'extensionArray'